build/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN env
 COPY build/_output/bin/handler.manager.linux-$TARGETARCH /usr/local/bin/manager
 
 RUN dnf install -b -y dnf-plugins-core && \
-    dnf install -b -y -x "*alpha*" -x "*beta*" nmstate && \
+    dnf install -b -y -x "*alpha*" -x "*beta*" nmstate-1.0.2 && \
```

Comment:
The PR says 1.1 but here I see 1.0. Which one is wrong?

Reply:
We have try 1.1.0 and it's failing so we pin at 1.0 but it's failing too.